roles/repository/tasks/restart.yml

```diff
@@ -0,0 +1,9 @@
+- name: Restart {{ svc }}
+  become: true
+  throttle: 1
+  vars:
+    svc: alfresco-content{% if groups.repository | length > 1 %}-monitored-startup{% endif %}
+  ansible.builtin.service:
+    name: "{{ svc }}"
+    state: restarted
+    daemon_reload: true
```

Comment:
not really needed?

```suggestion
```

Reply:
should not indeed but better to keep it i case notrification order  changes or we one day need to include just that file